src/main/java/formflow/library/inputs/AddressParts.java

```diff
@@ -1,10 +1,28 @@
 package formflow.library.inputs;
 
+/**
+ * Fields used to define an address fragment
+ */
 public enum AddressParts {
+  /**
+   * Street component of an address
+   */
   STREET_ADDRESS_1("StreetAddress1"),
+  /**
+   * Apartment, suite, or office number
+   */
   STREET_ADDRESS_2("StreetAddress2"),
+  /**
+   * City name
+   */
   CITY("City"),
+  /**
+   * The State value stored as the state code, a two character String, i.e. "IL", "CA", etc.
```

Comment:
When generating the docs, this line ends after the `i.e.` maybe because it takes the period as the end of a sentence?

  
<img width="779" alt="Screenshot 2023-10-10 at 1 08 04 PM" src="https://github.com/codeforamerica/form-flow/assets/9101728/5055b4e5-c2f1-497e-bffe-953bda3baf1e">


Later down the page the full sentence is displayed.
<img width="761" alt="Screenshot 2023-10-10 at 1 08 09 PM" src="https://github.com/codeforamerica/form-flow/assets/9101728/943a2059-a775-40f3-9d90-0006e938e3c2">


How I got to this page:
* Pulled down the PR
* Ran `./gradlew javadoc`
* Opened `build/docs/javadoc/index.html` in my browser
* Searched for `AddressParts`

Reply:
I've changed the copy to read ..., for example, "IL"....  This should address the i.e. issue